source/frontend/src/features/properties/filter/IPropertyFilter.ts

```diff
@@ -27,4 +29,5 @@ export const defaultPropertyFilter: IPropertyFilter = {
   longitude: '',
   page: undefined,
   quantity: undefined,
+  ownership: 'isCoreInventory,isPropertyOfInterest,isOtherInterest',
```

Comment:
isDisposed?

Reply:
Not part of the default set, per the story definition.